posydon/visualization/combine_TF.py

```diff
@@ -13,119 +13,134 @@
-                   'Primary got stopped before central carbon depletion',
-                   'Secondary got stopped before central carbon depletion',
-                   'Primary enters pair-instability regime',
-                   'Secondary enters pair-instability regime',
-                   'Primary enters pulsational pair-instability regime',
-                   'Secondary enters pulsational pair-instability regime',
-                   'offcenter neon ignition for primary',
-                   'offcenter neon ignition for secondary',
-                   'envelope_mass_limit',
-                   'gamma_center_limit',
-                   'Reached TPAGB',
-                   'max_age']
+TF1_POOL_STABLE = [
+    'Primary has depleted central carbon',
+    'Secondary has depleted central carbon',
+    'Primary got stopped before central carbon depletion',
+    'Secondary got stopped before central carbon depletion',
+    'Primary enters pair-instability regime',
+    'Secondary enters pair-instability regime',
+    'Primary enters pulsational pair-instability regime',
+    'Secondary enters pulsational pair-instability regime',
+    # 'offcenter neon ignition for primary',
+    # 'offcenter neon ignition for secondary',
+    'envelope_mass_limit',
+    'gamma_center_limit',
+    # 'Reached TPAGB',
+    'max_age'
+    ]
 
 TF1_POOL_UNSTABLE = [
-    'Unstable',
     'overflow from L2 (D_L2) distance for q(=Macc/Mdon)>1, donor is star 1',
     'overflow from L2 (R_L2) surface for q(=Macc/Mdon)<1, donor is star 1',
     'overflow from L2 (R_L2) surface for q(=Macc/Mdon)>1, donor is star 1',
     'overflow from L2 (D_L2) distance for q(=Macc/Mdon)<1, donor is star 1',
     'overflow from L2 (R_L2) surface for q(=Macc/Mdon)<1, donor is star 2',
     'overflow from L2 (R_L2) surface for q(=Macc/Mdon)>1, donor is star 2',
-    'reached maximum mass transfer rate: 10.0d0',
     'Reached maximum mass transfer rate: 1d-1',
-    'Reached the critical mt rate',
     'Reached maximum mass transfer rate: Exceeded photon trapping radius',
     'Both stars fill their Roche Lobe and at least one of them is off MS',
-    'Terminate due to L2 overflow during case A']
-
-TF1_POOL_INITIAL_RLO = ['Initial RLOF',
-                        'forced_initial_RLO',
-                        'overflow from L1 at ZAMS',
-                        'Terminate because of overflowing initial model',
-                        'overflow from L2 point for q>1 at ZAMS',
-                        'overflow from L2 surface for q<1 at ZAMS',
-                        'overflow from L2 surface for q>1 at ZAMS',
-                        'ignored_no_BH',
-                        'ignored_scrubbed']
-
-TF1_POOL_ERROR = ['logQ_limit',
-                  'logQ_min_limit',
-                  'min timestep',
-                  'min_timestep_limit',
-                  'cluster timelimit',
-                  'reach cluster timelimit',
-                  'no termination code',
```

Comment:
I think we should keep something like this. To allow us to flag stuff which is not covered by the termination flags.